app/experimenter/experiments/tasks.py

```diff
@@ -184,6 +187,18 @@ def update_status(experiment):
             bugzilla.update_bug_resolution(experiment)
 
 
+def send_experiment_ending_emails(experiment):
+    # send experiment ending soon emails if end date is 5 days out
+    if experiment.end_date - date.today() <= timedelta(days=5):
+        if not ExperimentEmail.objects.filter(
+            experiment=experiment, type=ExperimentConstants.EXPERIMENT_ENDS
```

Comment:
Not entirely sure if I understand correctly, we're filtering to see an ending email was sent for a given experiment?
Would we ever extend the timeline for an experiment? if so, would we modify the existing experiment entry with a new end date or would we have a new experiment entry for the extension. If it's the first case, would we want to send another ending email for the extension? Not exactly what the workflow is like and I hope I explained my questions clearly 😬

Reply:
@tiftran I think I understand your question. The question is, what happens when we send an email 5 days ahead, but then we end up extending the experiment far into the future, and you are wondering if we should be sending another email in that case? I am talking it over with Shell... she seems to think that it's so uncommon to do an extension that she doesn't see needing to build in another... but I am going to mull it over more.

Reply:
@tiftran, after talking it over with Shell and Jared, I think it seems best to limit this PR to just sending the first email.

Reply:
Very good point though @tiftran that will probably start coming up when we do the thing where experiments are editable while they're live.